docs/Config.md

```diff
@@ -88,6 +88,9 @@ git:
     # displays the whole git graph by default in the commits panel (equivalent to passing the `--all` argument to `git log`)
     showWholeGraph: false
   skipHookPrefix: WIP
+  # the main branches that other branches are based off and merged back to. Used to
+  # decide whether the shas of commits in the commits panel should be yellow or green.
```

Comment:
I'm not good at righting texts like this, feel free to replace it with a better one.